main.bundle.js

```diff
@@ -151,8 +154,13 @@
 	  });
 	};
 
+	var missingFoodField = function missingFoodField(name) {
+	  if (name === "") $('#missing-name-alert').append("Please enter a food name");
```

Comment:
If you have the if block in the event listener - you don't need it here as well
However, I like the idea of the return, in the event listener, kicking you out, and not posting the food.  Do we know if the rails backend has validations on the foods table?